### DISCUSSION TITLE
Considering ExtPay but how do I know if a customer is from EU (because of VAT)?

### DISCUSSION THREAD

**Discussion by hckiang** [Glench/ExtPay#183] [General]
I'm located at EU and I understand that ExtensionPay doesn't support Stripe's tax functionality yet. Let's say I sell for USD $10 no matter where they're from, i.e., I give up collecting VAT dynamically according to country...

Then I'll still absolutely need to know who is from EU and who is not because once a year I'll need to tell Uncle Sam how much revenue is VAT taxable--for each EU customer the tax agency wants 20-ish percents of sales value and for each non-EU customer he'll leave that alone.

I suppose you have the same problem in the US as well. Do you just tell Uncle Sam all your customers ought to pay VAT so at the end of the year you'll end up over-paying tax? Or how do you manage this?

**Comment by Glench** [comment]
ExtensionPay can support Stripe's tax functionality now in version 3.1 and above! I haven't enabled it yet, but I can turn on automatic tax collection. Or should it be an option you can enable yourself?

**Reply by hckiang** [reply]
Thanks for the implementation, Glench! I'm not familiar enough with the Stripe tax thing to know what does automatic tax collection entail: if you flip the switch on your side then will new transactions all come with extra tax? Either ways I think it's better to let users enable themselves at least for existing users because it might confuse the entire bookkeeping thing for people---I myself am using some flaky/semi-manual hacks to keep track of this and I suppose other users might be too. Or if you do, please send emails beforehand...
